{"tid":308609,"cid":26,"subCid":0,"title":"[知識分享] Cacti升級到1.x.x後出現的Error","createTime":"2017-07-04T10:13:52.000Z","updateTime":"2017-07-04T10:18:27.000Z","uid":9769,"like":0,"dislike":0,"uniUserReply":0,"replies":[{"pid":"f81bbde732d702fa333e1f3526de4b7e53480736","tid":308609,"uid":9769,"like":0,"dislike":0,"score":0,"citedBy":0,"replyTime":"2017-07-04T10:13:52.000Z","msg":"# After upgrade your Cacti to 1.x.x , you'll got this error:<br />\n<br />\nERROR: Your Cacti database login account does not have access to the MySQL TimeZone database. Please provide the Cacti database account &quot;select&quot; access to the &quot;time_zone_name&quot; table in the &quot;mysql&quot; database, and populate MySQL's TimeZone information before proceeding.<br />\n<br />\n<br />\n<img src=\"http://space.10gtechnology.com/mysql_upgrade_error.jpg\" data-thumbnail-src=\"https://i.lih.kg/thumbnail?u=http%3A%2F%2Fspace.10gtechnology.com%2Fmysql_upgrade_error.jpg&h=c465f5c6&s={SIZE}\" /><br />\n<br />\n<br />\n# Solution 解決方法<br />\n<br />\n# Login to mysql database as root 以root身份登入mysql<br />\n<br />\nmysql&gt; use mysql;<br />\nGRANT SELECT ON mysql.time_zone_name TO cacti@localhost IDENTIFIED BY 'Your_Cacti_Password';<br />\n<br />\nFLUSH privileges;<br />\n<br />\nquit<br />\n<br />\n<br />\n# Import timezone data 匯入timezone資料<br />\n[root@monitor01 ~]# mysql_tzinfo_to_sql /usr/share/zoneinfo | mysql -u root -p mysql<br />\n<br />\n<br />\n# Restart mysqld services 重啟mysqld服務<br />\n[root@monitor01 ~]# service mysqld restart"},{"pid":"23facab1fca96c27a5ba79396652c263df5ccda2","tid":308609,"uid":9769,"like":0,"dislike":0,"score":0,"citedBy":0,"replyTime":"2017-07-04T10:18:27.000Z","msg":"仲有, 更新之後, 有些plug-in可能會唔支援, 你要再裝過新版本D既plug-in."}]}